### ISSUE TITLE
Cryptsetup doesn’t use Inter

### ISSUE THREAD

**Issue by hanaral** [elementary/plymouth-theme#12]
### Problem

The drive encryption unlock screen uses Ubuntu’s font rather than Inter

**Comment by cassidyjames** [comment]
Photo for context. Similarly, the check disk screen when booting the installer also uses Ubuntu instead of Inter.

![PXL_20210204_000857146.jpg](https://user-images.githubusercontent.com/611168/106932967-9d6a7900-66d5-11eb-87f2-7c7ebbc4bedd.jpg)

![PXL_20210203_235928872.jpg](https://user-images.githubusercontent.com/611168/106932981-a22f2d00-66d5-11eb-889f-aa73fe30d9ba.jpg)

**Comment by meisenzahl** [comment]
I'm sorry I mixed up the issues. Sorry for that.

> plymouth will [fallback to "Sans 12"](https://gitlab.freedesktop.org/plymouth/plymouth/-/blob/c4ced2a2d70edea7fbb95274aa1d01d95928df1b/src/plugins/controls/label/plugin.c#L180) if no font description is provided.
> 
> But "Sans" should be resolved to "Arimo":
> 
> ```
> $ fc-match sans
> Arimo-Regular.ttf: "Arimo" "Regular"
> ```
> 
> So I'm not quite sure what's happening here...

_Originally posted by @meisenzahl in https://github.com/elementary/plymouth-theme/issues/13#issuecomment-773597056_

> @meisenzahl is it a matter of the font not being available from the OS? Do we need to do something special with the font to make it available?

_Originally posted by @cassidyjames in https://github.com/elementary/plymouth-theme/issues/13#issuecomment-773635400_

**Comment by meisenzahl** [comment]
@cassidyjames if I understand it correctly, `plymouth` is "compiled" into our `initramfs`. It's just a guess, but maybe in both cases we use only a minimal `initramfs` where Inter is not available.

@davidmhewitt could that, be the problem?

**Comment by davidmhewitt** [comment]
@meisenzahl Yes, correct, when the initramfs is updated, a series of "hook" scripts are ran to copy in the necessary files for the initramfs to run.

It's often better to view the source of a package from the Ubuntu repositories as Canonical often patch stuff. I've found this which looks like the culprit, and we're already patching plymouth, so probably not too much of a big deal to swap out the fonts here:

https://github.com/elementary/os-patches/blob/plymouth-focal-patched/debian/local/plymouth.hook#L116-L163

You could test locally by modifying `/usr/share/initramfs-tools/hooks/plymouth`.

**Comment by meisenzahl** [comment]
@davidmhewitt thanks for looking into this!
